todoman/ui.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, todo, databases, formatter):
         self._help_text = urwid.Text(
             '\n\nGlobal:\n'
             ' F1: Toggle help\n'
-            ' Ctrl-C: Cancel\n\n'
+            ' Ctrl-C: Cancel\n'
+            ' Ctrl-S: Save (only works if not a shell shortcut already)\n\n'
```

Comment:
I'd just bite the bullet and make it `ctrl s`. You can disable bash's usage of that combo and that's what I do anyway because I need that key for mutt.